Physics/HelicityFormalism/HelicityKinematics.cpp

```diff
@@ -119,7 +119,7 @@ HelicityKinematics::calculateHelicityAngles(const Event &Event,
 
     Recoil.Boost(DecayingState);
 
-    // rotate recoil so that recoil points in the z direction
+    // rotate recoil so that recoil points in the -z direction
```

Comment:
--> z-direction ?

Reply:
Afais `M_PI - Recoil.Theta()` puts the Recoil on the -z axis, while `-Recoil.Theta()` would put it on the z axis.